requirements.txt

```diff
@@ -8,6 +8,7 @@ seaborn
 tqdm
 pytest
 natsort
+nbmake
```

Comment:
We should have a separate `test_requirements.txt` for test only dependencies. It does not make much sense to pull `pytest`, `nbmake`, ... for all users who just want to run a notebook/script

Reply:
this is a good idea, thanks!